pkg/component/kubeapiserver/deployment.go

```diff
@@ -466,6 +466,9 @@ func (k *kubeAPIServer) computeKubeAPIServerArgs() []string {
 	for _, issuer := range k.values.ServiceAccount.AcceptedIssuers {
 		out = append(out, fmt.Sprintf("--service-account-issuer=%s", issuer))
 	}
+	if k.values.ServiceAccount.JWKSURI != nil && strings.TrimSpace(*k.values.ServiceAccount.JWKSURI) != "" {
```

Comment:
```suggestion
	if k.values.ServiceAccount.JWKSURI != nil {
```

We don't do this anywhere else, so let's also not do it here. We control the code that sets this field anyways.